packages/webpack-cli/lib/webpack-cli.js

```diff
@@ -67,11 +67,11 @@ class WebpackCLI {
     }
 
     // For babel/typescript
-    if (result.default) {
-      result = result.default;
+    if (result && typeof result === "object" && "default" in result) {
```

Comment:
Why we need `typeof result === "object"`?

Reply:
`result` can be of any type, thus we need to confirm it is object here to confirm we are handling babel/ts case and for usage of `in` operator we have to make sure `result` is `object`.